idb/postgres/internal/writer/writer_test.go

```diff
@@ -37,6 +37,44 @@ func setupPostgres(t *testing.T) (*pgxpool.Pool, func()) {
 	return db, shutdownFunc
 }
 
+// makeTx is a helper to simplify calling TxWithRetry
+func makeTx(db *pgxpool.Pool, f func(tx pgx.Tx) error) error {
+	return pgutil.TxWithRetry(db, serializable, f, nil)
+}
+
+type txnRow struct {
+	round    int
+	intra    int
+	typeenum idb.TxnTypeEnum
+	asset    int
+	txid     string
+	txnbytes []byte
+	txn      []byte
+	extra    string
+}
+
+// getTransactions is a test helper for checking the txn table.
+func txnQuery(db *pgxpool.Pool, query string) ([]txnRow, error) {
```

Comment:
it is simpler to pass `t *testing.T` and check all errors with one line instead of 3